src/components/v2/V2Project/V2ProjectReconfigureModal/V2ReconfigureModalTrigger.tsx

```diff
@@ -51,68 +38,6 @@ export default function V2ReconfigureFundingModalTrigger({
     reloadWindow()
   }
 
-  // Load queued FC data of project
-  const { data: queuedFundingCycleResponse } = useProjectQueuedFundingCycle({
-    projectId,
-  })
-
-  const [queuedFundingCycle] = queuedFundingCycleResponse ?? []
-
-  let effectiveFundingCycle: V2FundingCycle | undefined
-  // If duration is 0, use current FC (not queued)
-  if (!fundingCycle?.duration || fundingCycle?.duration.eq(0)) {
-    effectiveFundingCycle = fundingCycle
-  } else {
-    effectiveFundingCycle = queuedFundingCycle
-  }
-
-  const { data: queuedDistributionLimitData } = useProjectDistributionLimit({
-    projectId,
-    configuration: effectiveFundingCycle?.configuration.toString(),
-    terminal: primaryTerminal,
-  })
-  const [queuedDistributionLimit, queuedDistributionLimitCurrency] =
-    queuedDistributionLimitData ?? []
-
-  const { data: queuedPayoutSplits } = useProjectSplits({
-    projectId,
-    splitGroup: ETH_PAYOUT_SPLIT_GROUP,
-    domain: effectiveFundingCycle?.configuration?.toString(),
-  })
-
-  // Load queued FC data (if it exists) into redux
-  useEffect(() => {
-    if (queuedDistributionLimitData) {
-      dispatch(
-        editingV2ProjectActions.setDistributionLimit(
-          fromWad(queuedDistributionLimit),
-        ),
-      )
-      dispatch(
-        editingV2ProjectActions.setDistributionLimitCurrency(
-          queuedDistributionLimitCurrency.toNumber().toString(),
-        ),
-      )
-    }
-    if (effectiveFundingCycle?.duration) {
-      dispatch(
-        editingV2ProjectActions.setDuration(
-          effectiveFundingCycle?.duration.toNumber().toString(),
-        ),
-      )
-    }
-    if (queuedPayoutSplits) {
-      dispatch(editingV2ProjectActions.setPayoutSplits(queuedPayoutSplits))
-    }
```

Comment:
All of this removed code appears to already be in initialEditingData.

For example, this code is also over here: https://github.com/jbx-protocol/juice-interface/blob/82e92d3de4431c0b48be5ae00ef6ea91e4fe6ba9/src/components/v2/V2Project/V2ProjectReconfigureModal/hooks/initialEditingData.ts#L179-L182